specification/baremetalinfrastructure/resource-manager/Microsoft.BareMetalInfrastructure/preview/2020-08-06-preview/baremetalinfrastructure.json

```diff
@@ -0,0 +1,890 @@
+      "description": "List of AzureBareMetal operations"
+    },
+    "Operation": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the operation being performed on this particular object. This name should match the action name that appears in RBAC / the event service."
+        },
+        "display": {
+          "$ref": "#/definitions/Display",
+          "description": "Displayed AzureBareMetal operation information"
+        },
+        "isDataAction": {
+          "description": "indicates whether an operation is a data action or not.",
+          "type": "boolean"
+        }
+      },
+      "description": "AzureBareMetal operation information"
+    },
+    "Result": {
+      "description": "Sample result definition",
+      "properties": {
+        "sampleProperty": {
+          "type": "string",
+          "description": "Sample property of type string"
+        }
+      }
+    },
+    "Display": {
+      "type": "object",
+      "properties": {
+        "provider": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly form of the resource provider name. This form is also expected to include the publisher/company responsible. Use Title Casing. Begin with \"Microsoft\" for 1st party services."
+        },
+        "resource": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly form of the resource type related to this action/operation. This form should match the public documentation for the resource provider. Use Title Casing. For examples, refer to the “name” section."
+        },
+        "operation": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly name for the operation as shown to the user. This name should be concise (to fit in drop downs), but clear (self-documenting). Use Title Casing and include the entity/resource to which it applies."
+        },
+        "description": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The localized friendly description for the operation as shown to the user. This description should be thorough, yet concise. It will be used in tool-tips and detailed views."
+        },
+        "origin": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The intended executor of the operation; governs the display of the operation in the RBAC UX and the audit logs UX. Default value is 'user,system'"
+        }
+      },
+      "description": "Detailed BareMetal operation information"
+    },
+    "ErrorResponse": {
+      "description": "Error response.",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorDefinition",
+          "description": "The error details."
+        }
+      }
+    },
+    "ErrorDefinition": {
```

Comment:
Same as the common type feedback.

Reply:
I can't find an example of this one. I tried to replace the references of `#/definitions/ErrorResponse` with `"../../../../../common-types/resource-management/v1/types.json#/parameters/ErrorResponse"`, but I got the error message 
```
FATAL: Error: Error retrieving '$.parameters.ErrorResponse' (Error: Trying to retrieve 'ErrorResponse' from mapping that contains no such key)
Process() cancelled due to exception : Error retrieving '$.parameters.ErrorResponse' (Error: Trying to retrieve 'ErrorResponse' from mapping that contains no such key)
  Error: Error retrieving '$.parameters.ErrorResponse' (Error: Trying to retrieve 'ErrorResponse' from mapping that contains no such key)
```

Reply:
nevermind, I see my error here